### PR TITLE
Update tool - add keep_longest_isoform to agat

### PR DESCRIPTION
**Pull request by tbrown91**
One more program available in agat that we are in need of in order to create our genome reports. Tests were looking a little funky as I'm not sure if there are different output formats given by agat on different devices. For me Test14 was failing - which I don't think I touched.

### CODE REVIEW

File: tools/agat/agat.xml [hunk 1]
```diff
@@ -136,12 +136,19 @@
                 #end if
                 -pe $tool.prot_existence
                 -o galaxy_functional_annot
+        #else if $tool.selector == 'keep_longest_isoform'
+            @input_annotation_single@
+            agat_sp_keep_longest_isoform.pl
+                --gff '$input_annotation'
+                --config '$agat_configfile'
+                --output ./output &&
+            cat ./output > '${annotation}'
```
Comment (bgruening) [inline tools/agat/agat.xml:145]:
```suggestion
                --output '${annotation}'
```

We can do that in the other places as well. This extra call of cat is not needed.

Comment (tbrown91) [reply tools/agat/agat.xml:145]: ok, I can change these

**Comment by bgruening** [comment]
Nearly perfect! Thanks Tom! One small hint, never work on the master/main branch in git. This will cause you few problems down the road.

**Review by bgruening — APPROVED** [review]

**Comment by tbrown91** [comment]
yes Sir

**Pull request merged**